app/jobs/candidates/registrations/placement_request_job.rb

```diff
@@ -0,0 +1,14 @@
+module Candidates
+  module Registrations
+    class PlacementRequestJob < ApplicationJob
```

Comment:
Feels like this Job belongs as part of the `Bookings` subapp ?

`Bookings::Candidates::PlacementRequestJob`

or

`Bookings::PlacementRequest::PlacementRequestJob`